src/CreateList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function CreateList (props: {list: Array<any>}) {
```

Comment:
This should probably be called "DisplayList" - you don't create it here, you display it.

Reply:
Agreed.